.github/workflows/neuron-ci.yml

```diff
@@ -21,7 +21,7 @@ on:
 
 env:
   BUILD_TYPE: Release
-  DESIRED_CMAKE_VERSION: 3.15.0
+  DESIRED_CMAKE_VERSION: 3.16.3
```

Comment:
@alexsavulescu why, out of curiosity?

Reply:
Just that something worked locally 3.16.3 vs 3.15.0 but I don't have the history anymore to go back and see why